apps/block_scout_web/lib/block_scout_web/templates/layout/_topnav.html.eex

```diff
@@ -183,6 +183,7 @@
             </svg>
           </button>
         <% end %>
+        <div id="amplitude_api_key" style="display: none; position:absolute;"><%= Application.get_env(:block_scout_web, :amplitude_api_key) %></div>
```

Comment:
remove

Reply:
this is required since the amplitude sdk read the api key from here
`const amplitudeApiKey = document.getElementById("amplitude_api_key")?.innerText`
I hide it with `display: none`

Reply:
oh good to know, thanks for clarifying!